keylime/crypto.py

```diff
@@ -19,58 +19,119 @@
 '''
 
 import base64
+import hmac
+import hashlib
+import os
+import secrets
+
+# Crypto implementation using python cryptography package
+from cryptography import exceptions
+import cryptography.hazmat.primitives.asymmetric
+from cryptography.hazmat.primitives.ciphers import ( Cipher, algorithms, modes )
+from cryptography.hazmat.primitives import (hashes,serialization )
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric.rsa import generate_private_key
+
+aes_block_size = 16
```

Comment:
Can this be set to `algorithms.AES.block_size` instead of hard-coded to '16'?

Reply:
No objections from me.